docs/save-restore.md

```diff
@@ -23,4 +23,29 @@ To save and restore a session in VisiData:
 2. Press `gq` to quit VisiData.
 3. Replay the cmdlog, on the commandline: `vd -p fn.vd`.
 
+# How to replay a cmdlog on a different filename
+
+1. Load the **CommandLog**:
+
+    a. Either open an already saved .vd with `vd foo.vd`.
+
+    *or*
+
+    b. Load the **CommandLog** for the current session with `Shift+D` (commands executed on current sheet only) or `g Shift+D` (commands executed in the entire session).
+
+2. Ensure your cursor is on the **sheet** column. Press `gs` to select all rows, and `gzd` to empty every cell in the sheet column.
+
+3. Save the **CommandLog** with `Ctrl+S`.
+
+4. Pass the desired file through the cmdline when replaying:
+
+~~~
+vd -p foo.vd data.tsv
+~~~
+
+If the **sheet** column in a **CommandLog** is blank, VisiData will refer to the active sheet. Upon startup, that will be the first sheet passed through the commandline. VisiData will load that entire sheet, before replay progresses.
+
+Important, for this to work, all movements between sheets must be explicit within the .vd. This means you need to use `q` or `Shift+S` or `ALT+SheetNumber` to move between sheets.
```

Comment:
Do not recommend Alt+SheetNumber since the specific shortcut is not guaranteed.  Shift+S is also unnecessary; they should use jump-sheet or quit-sheet (or rely on a previous open).